elvis.config

```diff
@@ -8,7 +8,14 @@
        ruleset => erl_files,
        rules => [{elvis_style, line_length, #{limit => 100,
                                               skip_comments => false}},
-		 {elvis_style, dont_repeat_yourself, #{min_complexity => 20}}
+                 {elvis_style,
+                  variable_naming_convention,
+                  #{regex => "^([_A-Z][0-9a-zA-Z]*)$" }},
+%%                provision for newer version of elvis_core which doesn't automagically
+%%                strip leading underscore
+%%                https://github.com/inaka/elvis_core/commit/70fd9d01fd076f4640da09fdfd53b310ec59a94c
+%%                  #{regex => "^_{0,2}([A-Z][0-9a-zA-Z]*)$"}},
```

Comment:
After some hesitation, I think the best way would be to simply state we use Elvis 0.3.2, and when 0.3.3. is out switch to it and modify config file accordingly.